core/src/main/scala/chisel3/package.scala

```diff
@@ -37,13 +39,48 @@ package object chisel3 {
       case bigint                => Builder.error(s"Cannot convert $bigint to Bool, must be 0 or 1"); Bool.Lit(false)
     }
 
-    /** Int to UInt conversion, recommended style for constants.
-      */
-    def U: UInt = UInt.Lit(bigint, Width())
+    /** Int to UInt conversion, recommended style for constants. */
+    def U: UInt = UInt.Lit(bigint, Width()) // scalastyle:ignore method.name
 
-    /** Int to SInt conversion, recommended style for constants.
-      */
-    def S: SInt = SInt.Lit(bigint, Width())
+    /** BigInt to UInt conversion followed by bit extract (bad style, deprecated) */
+    @chiselRuntimeDeprecated
+    @deprecated(
+      "Passing an Int to .U is usually a mistake: It does *not* set the width but does a bit extract. " +
+        "Did you mean .U(<arg>.W)? " +
+        "If you do want bit extraction, use .U.extract(<arg>)",
+      "3.5.3"
```

Comment:
```suggestion
      "Chisel 3.5.4"
```
You always need to include the name of the project in the version ([see ScalaDoc for `@deprecated` which says this](https://www.scala-lang.org/api/2.13.8/scala/deprecated.html)).

Also Chisel 3.5.3 has already been released so this is Chisel 3.5.4.